gst/nnstreamer/tensor_common.c

```diff
@@ -112,9 +112,16 @@ static GstCaps *
 _get_tensors_caps (const GstTensorsConfig * config)
 {
   GstCaps *caps;
+  tensor_format format;
 
   caps = gst_caps_from_string (GST_TENSORS_CAP_DEFAULT);
 
+  format = config->info.info[0].format;
```

Comment:
Question: why "format" is in TensorInfo? not in TensorsInfo? Does anyone know why it should be in TensorInfo?

Reply:
If it's better be in TensorsInfo, not TensorInfo, it should be dealt with another PR later.

Reply:
Considered multi tensors with mixed format (0:static / 1:flex / ...). I will check this after #3389 (removing tensor-config).

Reply:
If you want to support mixed formats, ```other/tensors,format=${type}``` won't work.

Reply:
related issue #3405

Reply:
Fix this on https://github.com/nnstreamer/nnstreamer/pull/3418